packages/polling-controller/src/AbstractPollingController.ts

```diff
@@ -24,45 +18,35 @@ export const getKey = (
  */
 // TODO: Either fix this lint violation or explain why it's necessary to ignore.
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export function AbstractPollingControllerBaseMixin<TBase extends Constructor>(
-  Base: TBase,
-) {
+export function AbstractPollingControllerBaseMixin<
+  TBase extends Constructor,
+  PollingInput,
```

Comment:
does this need to `extends Json` since we need it to be stringifiable?

Reply:
I was also wondering this.  We do want the input to be serializable, but i wasn't sure what constraints would be necessary to guarantee that.